lib/IconButton/IconButton.js

```diff
@@ -195,5 +210,16 @@ module.exports = kind(
 		if (inEvent.originator === this) {
 			this.bubble('onRequestScrollIntoView');
 		}
+	},
+
+	/**
+	* @private
+	*/
+	backgroundOpacityChanged: function (old) {
```

Comment:
Hi @suhlee, the main gain would be to avoid repetition and to generalize this behavior, so that we can both easily reuse the functionality, and make updates in a single location. After talking with @ryanjduffy, we decided against it for now, as the repeated functionality is fairly small and limited, and there would be a lot of complications abstracting the styling. For the controls in `VideoPlayer`, those are `IconButton` controls, so the functionality should already be present.